CAM/UseVehicleCamStuntSettingsThisUpdate.md

```diff
@@ -0,0 +1,13 @@
+---
+ns: CAM
+aliases: ["0x6493CF69859B116A", "_USE_STUNT_CAMERA_THIS_FRAME"]
+---
+## USE_VEHICLE_CAM_STUNT_SETTINGS_THIS_UPDATE
+
+```c
+// 0x6493CF69859B116A
+void USE_VEHICLE_CAM_STUNT_SETTINGS_THIS_UPDATE();
+```
+
+Sets a stunt camera every update.
```

Comment:
Better description would be `Modify a set of settings for the vehicle camera to prevent confusion when performing specific stunts, such as navigating inside pipes or similar scenarios.`